{{cookiecutter.project_slug}}/Makefile

```
@@ -162,6 +162,43 @@ prod-secrets: ## Substitute with secrets template with env variable and run kube
 	# append the sealedsecrets annotation to the generated secrets manifest
 	sed -i '/spec:/ {N; s/spec:\n/  annotations:\n    sealedsecrets.bitnami.com\/managed: "true"\nspec:\n/}' k8s/prod/secrets.yaml
 
+## Monitoring
+
+kube-prometheus-up:  ## Install kube-prometheus
+	helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
+	helm repo update
+	helm install kube-prometheus prometheus-community/kube-prometheus-stack --set grafana.sidecar.datasources.isDefaultDatasource=false,grafana.persistence.enabled=true,grafana.adminPassword=$(openssl rand -base64 24) --namespace monitoring --create-namespace
```

I don't think openssl will be available on all platforms (win, mac and linux) so why don't we ask the user to export the password as an environment variable:
```suggestion
	helm install kube-prometheus prometheus-community/kube-prometheus-stack --set grafana.sidecar.datasources.isDefaultDatasource=false,grafana.persistence.enabled=true,grafana.adminPassword=${GRAFANA_ADMIN_PASSWORD} --namespace monitoring --create-namespace
```

---

I see ... that makes sense.

---

Hey @rochecompaan this would mean the password would be hardcoded in the codebase ... Does that make sense? I guess developers would usually have `openssl` installed.

---

You must export the password as an environment variable before running the make command.